### ISSUE TITLE
[1.12.2] [B16] [mild irritation] no use for herba in alchemy :(

### ISSUE THREAD

**Issue by MalkContent** [Azanor/thaumcraft-beta#979]
Edit: This is issue is only regarding the crucible.
----
In the TC6.0 versions there still was the cobblestone -> mossy cobblestone recipe so you had a sponge for all the excess herba.
Right now it is borderline impossible to figure out any "clean" hedge alchemy recipe combinations with the crucible, because you almost always end up having to use something with herba and since herba has no use in alchemy as of now, you either throw in more quartz or let it out into the aura.
I'd really like any kind of alchemical recipe that uses herba in some capacity, because I always felt that figuring out a fitting combination or recipes was a fun part of crucible alchemy.

I can see why the Moss Stone/Mossy Stone Bricks recipe got removed (they vanilla recipes now), but how about some regular old dirt -> grass/mycelium/podzol or some more fancy stuff like leaves -> respective sapling, doubling mushrooms, turning ferns/grass into their double tall versions, poppies to roses, pink tulips to peonies. these recipes could contain any number of other essentia requirements, as long as I can sink the herba into it somehow :/

**Comment by Miral007** [comment]
If you want to get rid of it safe and fast, put a lamp of growth on your herba jar.

**Comment by MalkContent** [comment]
Doesn't work for the crucible. Updated the initial post.

**Comment by Miral007** [comment]
That's true, but I think the crucible is meant to be inefficient, then you move on to the smelter and thaumatorium.

**Comment by MalkContent** [comment]
Crafting with the crucible without creating waste essentia was always difficult, but it's much more so now. I'm not asking for easy mode, I'm just being like "woah there" now that you basically can't use any plants in crucible crafting without creating flux/using quartz.
Moving to the smelter and the thaumatorium is not so much more efficient as it is a looot easier and also automatable.

**Comment by Azanor** [comment]
Any suggestions?

**Comment by Cenotaffio** [comment]
maybe a process that use herba in order to multiply seeds?
for example put one seed in a crucible with 20 herba to obtain 2 seeds.
In my opinion is quite balanced: in early game allow players to get rid of exceeding herba while speeding up the process of creating a farm, or obtain a consistent amount of rare seeds.
in mid/late game can be still useful if you have mods like immersive engineering for oil production.

Another thing can be a way to produce magical related plants, like visroom.
Or maybe use herba and precantatio to turn normal wood in a magic counterpart (like oak wood to greatwood).

**Comment by Cenotaffio** [comment]
Another suggestion (i think in the almost-op realm) could be a special soil or totem that can make mushroom grown regardless of light level.
obviously also done with dark essentia.

**Comment by MalkContent** [comment]
I had this idea where it would just make seeds grow their plants, (insert WOOSH sound) right out of the crucible, because I thought that was a kind of funny and reminded me of the plant scientist villain from Darkwing Duck.
Same thing for saplings, but the tree shape would be custom, in a multi-pronged tentacly, warpy shape, and not instant, but very quickly grow it block by block, consuming herba for each block. At the end it does an instant WOOSH (insert breaking wool sound) where the leaves grow at the end \^\^ 

Bottom line, it's just funny in my head. It might be sort of useful in the early game, but it also makes it difficult to use more than 1 seed/sapling for their essence. (And also you'd probably drop the grown item right back into the crucible while mining a lot :x)

**Comment by gerbilfat** [comment]
There's some good suggestions in here. The crucible was always meant to be inefficient, but there's two hurdles that combine to make the crucible an Achilles heel for the budding thaumaturge. Many of the recipes are no longer clean combined with no way to deal with flux buildup in the environment (yet). The research for doing an alchemy lab is more expensive than it used to be. Players are relying on the crucible for a longer than in previous versions of Thaumcraft. It's been a problem in SMP where multiple people need to use the crucible. We're all afraid of a rift opening up in our woodland mansion.

Giving us more recipes to soak up extra essentia is one way to solve the problem. Another is to implement some way to clean aura, either by player action or by monster spawns, thunderstorms, etc. consuming the flux over time.

**Comment by MalkContent** [comment]
Maybe there could be some kind of fertilizer that could be crafted with herba (and maybe some other aspects). Or just a regular old duplication recipe for bonemeal. Fertilizer Items are very useful early on, and later in the game, they would still be relevant for other mods. For example Immersive Engineering Garden Cloches use Fertilizers.

Or maybe "Fertilizer Sponge" that you can soak with herba.

Ooor, and this might be not in line with the fluff, you could use a lot of herba (+ a little of fitting other aspects like Praecantatio and Metallum) to "grow back" durability on thaumium items. Y'know, since we don't have repair anymore (and because Mending was a mistake :x). This could be behind a very early warpy inspiration (as in "you get your first perma warp, also you scanned thaumium and plants at some point) and en route to void metal (which is the true "i grow back by myself"  metal), and offer a place to spend an Eldritch Observation :)

**Comment by MalkContent** [comment]
Or! A fertilizer liquid with buckets as catalist. The liquid would spread like water and after a couple of ticks, the source block would vanish and cause a small aoe bonemeal effect around all blocks touched by it and the connected non-source blocks. Could grow grass on dirt and moss on cobblestone/bricks. Could turn coarse dirt into podzol. Could turn dirt into mycelium if there's some flux in the air (seeing as mycelium has Vitium now).
When clicking plants directly with it, it could just apply a couple doses of bonemeal to that single plant, to get around the "i have to build a trench, so this liquid doesn't wash away my plants" issue.
It could be called "Liquid Life", and affect animals, too, and be a prerequisite for the Lamp of Fertility/Growth, and be in front of Liquid Death :)

**Comment by Geethebluesky** [comment]
Going in a different direction... Herba is the TC essence of plants, and what plant was most beloved in previous versions (besides Silverwood Saplings of course): Ethereal Blooms!
 
I would love to see the return of the Ethereal Bloom as a Herba sink, especially if Taint is supposed to be nasty + a normal part of the progression; at least, as an early-game method of moderating areas of Taint a bit until the player can figure out how to deal with the newbie mistake of crucible-dumping too early and creating Rifts... This could be an early stage of Flux-mitigating research, since Flux is such a big deal overall it seems normal there would be multiple ways of combating it. 

Perhaps Herba can be used to craft an antidote or preventative to Taint Poison.

**Comment by Azanor** [comment]
This issue was moved to Azanor/thaumcraft-suggestions#352
